runtime-modules/council/src/lib.rs

```diff
@@ -266,6 +306,15 @@ decl_event! {
 
         /// The candidate has set a new note for their candidacy
         CandidacyNoteSet(MembershipId, Vec<u8>),
+
+        /// The whole reward was paid to the council member.
+        RewardPayment(MembershipId, AccountId),
```

Comment:
Can we include what was paid out and what new debt is for this member, that way we don't need a separate event for partial or missing payment.

Reply:
Done